### ISSUE TITLE
Search Bug

### ISSUE THREAD

**Issue by JulianLaNeve** [Dogfalo/materialize#2259]
Hey guys, when using Meteor.js, the search appears to be broken. 
Code:
```html
<div class="col s8 offset-s2">
        <form>
            <div class="input-field">
                <input id="search" type="search" required>
                <label for="search"><i class="material-icons">search</i></label>
                <i class="material-icons">close</i>
            </div>
        </form>
    </div>
```
Screenshot:
<img width="1680" alt="screen shot 2015-11-07 at 3 34 38 pm" src="https://cloud.githubusercontent.com/assets/5252587/11017466/15326720-8565-11e5-9230-557c0acc378c.png">

I'm trying to add it outside of the navbar, but that shouldn't make a difference.
Thanks

**Comment by JulianLaNeve** [comment]
Also, I'm using Meteor.js as a front/back end if that makes a difference.

**Comment by acburst** [comment]
This was fixed in #1669. If you are using 0.97.0 meteor the bug will still be there. We are going to release a new version soon where this will be fixed.
